core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -419,4 +424,29 @@ public void testCanReadOldCompressedManifestFiles() throws Exception {
     List<FileScanTask> tasks = Lists.newArrayList(reloaded.newScan().planFiles());
     Assert.assertEquals("Should scan 1 files", 1, tasks.size());
   }
+
+  @Test
+  public void testConcurrentAppend() throws Exception {
```

Comment:
I's fine to add more tests, but please copy the `testConcurrentFastAppends` test from JDBC into this suite. We have validated that it catches quite a few problems so we should reuse it.

Reply:
done

Reply:
Can you rename the method to match the source?

Reply:
Actually, can you copy the original without modification other than to set up the catalog? It is too heavily modified from the original version.

Reply:
actually there are no significant changes, just making some constant in variable ....and changing the values of those constant, 

the major reason to change to larger values is to make the issue easier to reproduce (it is hard to see the failure of committing with only 2 threads as set in JDBC test)